### ISSUE TITLE
final peer review--yc2576

### ISSUE THREAD

**Issue by YijiaCui** [tanmaybansal01/orie4741-f19#9]
This project aims at developing a model that predicts the relationship between geographical region within the US and its risk of mass shooting. They want to show that lax gun laws are a predictor for mass shootings, and that zip codes also correlate with mass shootings. Since their topic does not have enough data, they created their own dataset by aggregating and cleaning the desired data from six different datasets. Their dataset includes information about incident details, shooter’s personal information, regional demographic information, and applicable gun laws for each mass shooting in the US after 2010. 

Likes:
1. They really paid attention and effort in creating their own dataset. They considered what features might be useful and significantly related to their target. Even though they cannot find the desired dataset online, they made their own dataset. 
2. They paid attention to overfitting. Given the size of their dataset, this is a significant step.
3. They compared different models and loss functions. By doing so, they can choose the most accurate model and therefore their model and predictions are reliable. 

Areas to improve:
1. As they also mentioned in the “Looking forward” part, it’s better that they analysis correlated features since there are 2 out of 3 main features that are race-associated in their conclusion.
2. It would be more meaningful if they can also analyze the pattern of mass shooting in those areas with high numbers of shooting.
3. it would be better if they can predict the likelihood or patterns of shootings. 

Overall, this is a meaningful project, and all works are amazingly done.
